Dockerfile

```diff
@@ -51,17 +66,48 @@ RUN yum -y install \
     yum clean all && \
     rm -rf /var/cache/yum
 
+# Add hadoop repo. and install fetchdt
+ADD ./repos/hdp7-stable.repo /etc/yum.repos.d/hdp7-stable.repo
+RUN yum -y install \
+      hadoop-fetchdt && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+# Add openstack repo. and install kubernetes-client, helm
+ADD ./repos/openstackclients7-queens-stable.repo /etc/yum.repos.d/openstackclients7-queens-stable.repo
+RUN yum -y install \
+      kubernetes-client \
+      helm && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
 # Upgrade pip package manager
 RUN pip3.6 install --upgrade pip
 
 # ----- Install JupyterHub ----- #
+
+# Install JupyterHub dependencies for postres db support, pycurl over https and cryptography for auth state
+RUN yum install -y \
+      libcurl-devel && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+
+RUN PYCURL_SSL_LIBRARY=nss pip install \
+    statsd==$STATSD_VERSION \
+    psycopg2==$PYPOSTGRES_VERSION \
+    cryptography==$CRYPTOGRAPHY_VERSION \
+    pycurl==$PYCURL_VERSION
+
+# Install configurable-http-proxy
+RUN npm install -g configurable-http-proxy@$CHP_VERSION
+
 # Install JupyterHub with upstream authenticators and spawners
-RUN pip install jupyterhub$JUPYTERHUB_VERSION
-RUN npm install -g configurable-http-proxy
-RUN mkdir -p /var/log/jupyterhub
+RUN pip install \
+    jupyterhub==$JUPYTERHUB_VERSION \
+    jupyterhub-ldapauthenticator==$LDAPAUTHENTICATOR_VERSION \
+    jupyterhub-dummyauthenticator==$DUMMYAUTHENTICATOR_VERSION 
```

Comment:
do we need this?

Reply:
dummyauthenticator is required for development mode, ldapauthenticator was already there, @ebocchi was using it for HelixNebula project ?

Reply:
The ldap authenticator is the one used by default in sciencebox. I was just talking about the the dummy one. If you think is relevant for dev, let's keep it.